src/example/App/VariableText.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import createReactClass from 'create-react-class';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
```

Comment:
FYI: [`react/lib/ReactComponentWithPureRenderMixin`]([url](url)) will remove in React v16
See also https://github.com/satya164/react-navigation-addons/issues/5